From 2ea16be20271dae61156a157b9fabd542b328387 Mon Sep 17 00:00:00 2001
From: Dmitry Safonov <dima@arista.com>
Date: Thu, 10 May 2018 19:14:41 +0100
Subject: [PATCH 1/8] criu: Remove PAGE_IMAGE_SIZE

It's unused since commit fd3f33f5d23a ("headers: image.h -- Drop unused
entries"), so let's remove it completely.

Signed-off-by: Dmitry Safonov <dima@arista.com>
Signed-off-by: Andrei Vagin <avagin@virtuozzo.com>
---
 criu/crtools.c       | 1 -
 criu/include/image.h | 5 -----
 2 files changed, 6 deletions(-)

diff --git a/criu/crtools.c b/criu/crtools.c
index 5cf53d8a0..4ff7a9f24 100644
--- a/criu/crtools.c
+++ b/criu/crtools.c
@@ -356,7 +356,6 @@ int main(int argc, char *argv[], char *envp[])
 
 #undef BOOL_OPT
 
-	BUILD_BUG_ON(PAGE_SIZE != PAGE_IMAGE_SIZE);
 	BUILD_BUG_ON(CTL_32 != SYSCTL_TYPE__CTL_32);
 	BUILD_BUG_ON(__CTL_STR != SYSCTL_TYPE__CTL_STR);
 	/* We use it for fd overlap handling in clone_service_fd() */
diff --git a/criu/include/image.h b/criu/include/image.h
index d9c4bdbc8..d576f5a13 100644
--- a/criu/include/image.h
+++ b/criu/include/image.h
@@ -12,11 +12,6 @@
 #include "log.h"
 #include "common/bug.h"
 
-#ifdef _ARCH_PPC64
-#define PAGE_IMAGE_SIZE	65536
-#else
-#define PAGE_IMAGE_SIZE	4096
-#endif /* _ARCH_PPC64 */
 #define PAGE_RSS	1
 #define PAGE_ANON	2
 
-- 
2.17.0

